packages/svelte/src/lib/Dotlottie.svelte

```diff
@@ -98,14 +98,13 @@
 	});
 
 	$effect(() => {
-		console.log(speed);
-		if (dotLottie && dotLottie.isLoaded && typeof speed == 'number') {
+		if (dotLottie && typeof speed == 'number' && dotLottie.isLoaded) {
```

Comment:
changed order of conditions, cause `isLoaded` is not reactive which blocking svelte from getting to next reactive object which is `speed`.

thanks to https://github.com/sveltejs/svelte/discussions/14517#discussioncomment-11446142

Reply:
@moonlitgrace Sorry for the late reply. Is the PR ready for review?

Before that, I'm not sure if Svelte 5 is backward compatible with Svelte 4. Is that the case? If it is, we’ll also need to update the peer dependencies of the dotlottie-svelte package to include both v4 and v5 of Svelte.